docs/en/04_Changelogs/4.1.0.md

```diff
@@ -0,0 +1,87 @@
+# 4.1.0
+
+## Introduction
+
+This version introduces many breaking changes, which in most projects can be managed through a combination
```

Comment:
> This version introduces many breaking changes...

... is a worrying way to start a minor release changelog 😛. Does this actually break anything if you `composer update` 4.0.x to 4.1.x _without_ opting into the `public/` directory?

Reply:
Oh dear I've copied and pasted the 4.0.0.md changelog intro and didn't modify the intro again.